lib/dovico/model/time_entry.rb

```diff
@@ -54,7 +54,9 @@ def self.search(start_date, end_date)
 
     def self.batch_create!(assignments)
       api_assignements = assignments.map(&:to_api)
-      ApiClient.post(URL_PATH, body: api_assignements.to_json)
+      if api_assignements.count > 0
+        ApiClient.post(URL_PATH, body: api_assignements.to_json)
+      end
```

Comment:
should we output an error message if assignments are empty?

Reply:
No sure, because if happens when the user set no time entries to be generated.

So it's expected for the user to have 0 entries to be posted.